Packages/com.unity.inputsystem/InputSystem/Plugins/PlayerInput/PlayerInput.cs

```diff
@@ -1068,14 +1204,20 @@ public static PlayerInput Instantiate(GameObject prefab, int playerIndex = -1, s
         /// automatically pair one or more specific devices to the newly created player.
         /// </summary>
         /// <param name="prefab">A player prefab containing a <see cref="PlayerInput"/> component in its hierarchy.</param>
-        /// <param name="playerIndex"></param>
-        /// <param name="controlScheme"></param>
-        /// <param name="splitScreenIndex"></param>
-        /// <param name="pairWithDevices"></param>
-        /// <returns></returns>
+        /// <param name="playerIndex">Player index to instantiate.</param>
+        /// <param name="controlScheme">Control scheme to activate.</param>
+        /// <param name="splitScreenIndex">Which split screen to instantiate on.</param>
+        /// <param name="pairWithDevices">Which devices to limit pairing to.</param>
+        /// <returns>Newly created PlayerInput component</returns>
```

Comment:
Missing full stop

Reply:
Thank you. I'll pay more attention to these. Adding in next commit